protractor.conf.js

```diff
@@ -29,7 +29,15 @@ exports.config = {
     version: '',
     platform: 'ANY',
     shardTestFiles: true,
-    maxInstances: process.env.CHROME_INSTANCES
+    maxInstances: process.env.CHROME_INSTANCES,
+    download: {
+      prompt_for_download: false
+    },
+    chromeOptions: {
+      prefs: {
+        'download.default_directory':'./web/tmp/downloads'
```

Comment:
@ezequielc yes. `web/tmp` is guaranteed to exist because of the build, and I've tested that the `downloads` folder is created automatically. Yes, I think we should switch it to use `/tmp`, I didn't even realize. Thanks.